### ISSUE TITLE
default footer

### ISSUE THREAD

**Issue by tibfox** [3speak/bugs_and_suggestions#4]
It would be cool if the user would have the possibility to set a default footer for the post. So the extra step to copy and paste one would become unnecessary.

**Comment by wehmoen** [comment]
You mean like a signature?

**Comment by tibfox** [comment]
Yeah it is basically more text and elements including html than a simple
signature. That's why I call it footer.
Just look at my posts, all the banners and so on are the same for every
post over 2-3 months.

Nico Wehmöller <notifications@github.com> schrieb am Fr., 3. Mai 2019,
00:47:
